### PR TITLE
[TT-5570]

### PR DESCRIPTION
**Pull request by AMM55**
Creating the Tyk Operator pages in Hugo

### CODE REVIEW

File: tyk-docs/content/tyk-stack/tyk-operator/tyk-operator.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Tyk Operator is an open-source agent deployed to your Kubernetes cluster. It actively detects configuration drift between the API configurations on Gateway (the actual state) and the manifest (the desired state) to reconcile it. Therefore, the manifests become the source of truth for your API configurations.
+
+Tyk Operator also offers an Ingress Controller, which dynamically manages Tyk ApiDefinition resources as per the ingress specification. This way your Tyk Gateway is configured as a drop-in replacement for a standard Kubernetes Ingress, providing Security, Traffic Control and Limiting for external access to the services in your Kubernetes cluster.
+
+Tyk Operator works with the v3+ Open Source Tyk Gateway, our full self-managed Gateway, and Tyk Dashboard installation.
+
+## What can you do with Tyk Operator?
+
+- Kubernetes Ingress Controller
```
Comment (carolwkc) [inline tyk-docs/content/tyk-stack/tyk-operator/tyk-operator.md]: Link: https://github.com/TykTechnologies/tyk-operator/blob/master/docs/ingress.md

File: tyk-docs/content/tyk-stack/tyk-operator/tyk-operator.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- Manage API Definitions
```
Comment (carolwkc) [inline tyk-docs/content/tyk-stack/tyk-operator/tyk-operator.md]: Link: https://github.com/TykTechnologies/tyk-operator/blob/master/docs/api_definitions.md

File: tyk-docs/content/tyk-stack/tyk-operator/tyk-operator.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+- Manage Security Policies*
```
Comment (carolwkc) [inline tyk-docs/content/tyk-stack/tyk-operator/tyk-operator.md]: Link: https://github.com/TykTechnologies/tyk-operator/blob/master/docs/policies.md

File: tyk-docs/content/tyk-stack/tyk-operator/tyk-operator.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+- Manage Developer Portal
```
Comment (carolwkc) [inline tyk-docs/content/tyk-stack/tyk-operator/tyk-operator.md]: Link: https://github.com/TykTechnologies/tyk-operator/blob/58b0e350cd73250c35faeaefab55627cc4bd2e61/config/samples/portal/apidescription.yaml

**Review by craigtyk — APPROVED** [review]
Nice work, Operator docs looking great. OK to publish

**Review by marksou — APPROVED** [review]

**Pull request merged**